extract_zim/Makefile

```diff
@@ -12,7 +12,7 @@ extract_zim: bin/extract_zim
 	cp -f bin/extract_zim extract_zim
 
 bin/extract_zim: cargo
-	cargo install --force --bin extract_zim --rev a283151105ab4c1905d7f5cb56fb8eb2a854ad67 --git https://github.com/dignifiedquire/zim --root .
+	cargo install --force --bin extract_zim --rev c8c0e68d1bd96c4b5c45d27eafd778630ff42edf --git https://github.com/kanej/zim --root .
```

Comment:
This is blocked by https://github.com/dignifiedquire/zim/pull/9, right?

Reply:
The html in the zim files link to jpeg images with 'jpg', 'jpeg' and most delightfully 'JPG'. In our injected js we clean up the file ending to all be 'jpg'. I went with 'jpg' because it is be far the most common, so we reduce the number of 404s on initial page load that the injected js then corrects.

Reply:
Is it possible that upstream Wikipedia might be interested in cleaning up these file extensions in an automated way, if that is practical for them?

Reply:
I believe Wikipedia has open issues about it already, for example:
https://phabricator.wikimedia.org/T247503
https://phabricator.wikimedia.org/T6421

Sticking to `.jpg` sounds like a future-proof choice :+1: